### ISSUE TITLE
Problem with hiding post if translation is not available

### ISSUE THREAD

**Issue by dblenkus** [qTranslate-Team/qtranslate-x#403]
Hi,

I'm trying to hide posts that are not available in selected language. I've selected "Hide Content which is not available for the selected language.", but posts are still shown as "Sorry, this entry is only available in [language]."

Is there an option to achieve the desired functionality?

Thanks,
Domen

**Comment by johnclause** [comment]
Please, describe what is a "desired"? They are hidden in search, but when you access it directly, how do you want them to be hidden? 404 error? Probably not. Then what?

**Comment by dblenkus** [comment]
Thanks for your answer!

Sorry if my explanation was not clear enough. The "desired" behavior is to hide such posts from first page and be able to access them only with exact url.

[Here](https://www.ajdas.com/en/page/2) is an example. You can se that fourth post is not translated to english, but appears anyway.

Hope that now it is clearer what is the problem.

Best,
Domen

**Comment by johnclause** [comment]
I am not sure what is going on, indeed, it should not be there. If you switch the theme to a standard one like Twenty Sixteen, is it still listed?

**Comment by pidal** [comment]
Did you find a solution for this? @dblenkus I'm facing the same problem with a custom theme. Untranslated posts are displayed in the main loop of wp with the qTranslate message "Sorry, this entry..."

**Comment by dblenkus** [comment]
No, I haven't found the solution. But it is not so annoying for me, as I only had one post in single language. 

But never the less it would be great if the problem is solved.
